.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "8"
+  - "6"
+  - "4"
```

Comment:
Not sure I follow, does a dep require >=6?

Reply:
Otherwise I'm hesitant to remove 4 until it's not LTS anymore.

Reply:
hmm not sure why but noticed that on node 4 tests are failing: 
<img width="694" alt="screen shot 2017-09-02 at 10 37 08 pm" src="https://user-images.githubusercontent.com/5240221/30000869-d399a4cc-902f-11e7-8c15-dd7e154ff45e.png">

Reply:
Looks like this is due to [jest-json-schema](https://github.com/americanexpress/jest-json-schema) using [default argument values](https://github.com/americanexpress/jest-json-schema/blob/v1.2.0/index.js#L20) which aren't handled by Nodejs v4, but nor is the "engines" field supplied.

Reply:
also [destructuring](https://github.com/americanexpress/jest-json-schema/blob/master/matchers/toMatchSchema.js#L17) and [let](https://github.com/americanexpress/jest-json-schema/blob/master/matchers/toMatchSchema.js#L28)
but [so does jest-image-snapshot](https://github.com/americanexpress/jest-image-snapshot/blob/master/__tests__/src/index.spec.js#L30)

Reply:
While I'd rather keep 4 in the list while it's LTS, I can begrudgingly remove it from `.travis.yml` if adding `"engines": { "node": ">=6" }` is acceptable.

Reply:
alright, I'm a bit off
v4 is in Maintenance, not Active (as of April) https://github.com/nodejs/LTS
As [this package was published <1mo ago](https://github.com/americanexpress/jest-image-snapshot/releases/tag/v1.0.0) it would seem likely this would be used in projects using the Active LTS version(s), today v6.

Reply:
Okay I will open PR to add engines and merge this.